react-native/react/pinentry/index.render.desktop.js

```diff
@@ -14,6 +14,10 @@ export default class PinentryRender extends Component {
     }
     for (const feature in this.props.features) {
       this.state.features[feature] = this.props.features[feature].defaultValue
+
+      if (feature === 'showTyping') {
+        this.state.showTyping = this.props.features[feature].defaultValue
```

Comment:
actually this should be the constructor, i think it moved around, but i changed it back